docs/src/examples/components/Popup/BestPractices/PopupBestPractices.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+
+const doList = [
+  {
+    key: 'one',
+    content: 'Use popupFocusTrapBehavior if the focus needs to be trapped inside of the Popup.',
+  },
+  {
+    key: 'two',
+    content:
+      "If Popup's content is lazy loaded and focus needs to be trapped inside - make sure to use state change to trigger componentDidUpdate, so the focus can be set correctly to the first tabbable element inside Popup or manually set focus to the element inside once content is loaded.",
```

Comment:
Are both triggering `componentDidUpdate` and manually setting focus both equally good options or is one more preferable than another? Is there a reason to use one over the other?

Reply:
I think that here you need to try and see which version works better for a specific case. We'll add more information to the details of Popup as part of #1572 work.